examples/vad_transcriber/wavTranscription.md

```diff
@@ -15,6 +44,18 @@ Setup your environment
 (venv) ~/Deepspeech/examples/vad_transcriber$ pip3 install -r requirements.txt
 ```
 
+Fedora
+
+```
+~/Deepspeech$ sudo dnf install python-virtualen
```

Comment:
typo? Shouldn't this be `python-virtualenv` ?

Reply:
Yes, will fix in the next commit